pages/index.tsx

```diff
@@ -87,7 +88,15 @@ const LandingPage = ({
   const { data: mkrInChief } = useTokenBalance(Tokens.MKR, chiefAddress);
   const { data: hat } = useHat();
   const { data: mkrOnHat } = useMkrOnHat();
-  const { account } = useAccount();
+  const { account, voteDelegateContractAddress, voteProxyContractAddress } = useAccount();
+  const { data: votedProposals, mutate: mutateVotedProposals } = useVotedProposals();
+
+  const address = voteDelegateContractAddress || voteProxyContractAddress || account;
+
+  // revalidate votedProposals if connected address changes
+  useEffect(() => {
+    mutateVotedProposals();
```

Comment:
doesn't it revalidate automatically inside the hook if the address changes? sounds like something the hook itself should handle